### DISCUSSION TITLE
The color is not as good as before

### DISCUSSION THREAD

**Discussion by UallenQbit** [saadeghi/daisyui#2583] [Q&A] [answered]
The colors of the new version feel really ugly. The colors of the last 3.0 version still look good. The new version is too dark....
Especially the green color of success is too dark....

new version
![image](https://github.com/saadeghi/daisyui/assets/72503738/a39f4f92-a759-43bc-a3c6-70e03400bf36)

old version
![image](https://github.com/saadeghi/daisyui/assets/72503738/aab33ca8-45f8-4f22-b92f-1c7410b7da7c)

**Comment by saadeghi** [comment, marked as answer]
Use theme generator to change the colors the way you like:  
https://daisyui.com/theme-generator/

Here's how to customize built-in themes:  
https://daisyui.com/docs/themes/#-7

The built-in themes are optional to use and the color changes from v3 to v4 was made to increase the contrast and accessibility.
